src/Presentation/Camera/Fetching/Types/TPLongitude.cs

```diff
@@ -0,0 +1,20 @@
+namespace Presentation.Camera.Fetching.Types
+{
+    using System.Diagnostics.CodeAnalysis;
+    using Business.Camera.Fetching.Models;
+    using global::GraphQL.Types;
+    using Style;
+
+    [SuppressMessage(Category.Default, Check.CA1724, Justification = Reason.Readability)]
+    public class TPLongitude : InputObjectGraphType<PLongitude>
+    {
+        public TPLongitude()
+        {
+            Name = "LongitudeFilter";
+            Description = "This will be used for filtering the cameras by longitude";
+
+            Field(longitude => longitude.LesserEqualThan, true).Description("Cameras will have the longitude lesser or equal than this value.");
```

Comment:
Also here, please move the description to the next line and space between the fields.

Reply:
done